examples/vacationsApprovalServerless/README.md

```diff
@@ -0,0 +1,89 @@
+# Vacation Approval App example using Serverless Framework
+
+With Smartmate you have the ability to call custom endpoints in order to have manage custom logic outside the Smartmate environment.
+
+This example explains how to setup a Smartmate App using Serverless Framework.
+
```

Comment:
Remove empty lines that are not needed, and add lines where needed. For example: before the next line `Note`, there are 2 empty lines, whereas, after that line, there's no empty line before the next title.